.gitignore

```diff
@@ -2,3 +2,18 @@
 *.user
 *.dll
 *.pdb
+*.exe
+*.suo
+/packages/TestStack.BDDfy.4.3.2/TestStack.BDDfy.4.3.2.nupkg
+/packages/System.Threading.Tasks.Extensions.4.3.0/ThirdPartyNotices.txt
+/packages/System.Threading.Tasks.Extensions.4.3.0/System.Threading.Tasks.Extensions.4.3.0.nupkg
+/packages/System.Threading.Tasks.Extensions.4.3.0/lib/portable-net45+win8+wp8+wpa81/System.Threading.Tasks.Extensions.xml
+/packages/System.Threading.Tasks.Extensions.4.3.0/lib/netstandard1.0/System.Threading.Tasks.Extensions.xml
+/packages/System.Threading.Tasks.Extensions.4.3.0/dotnet_library_license.txt
+/packages/Npgsql.3.2.3/Npgsql.3.2.3.nupkg
+/packages/Npgsql.3.2.3/lib/netstandard2.0/Npgsql.xml
+/packages/Npgsql.3.2.3/lib/netstandard1.3/Npgsql.xml
+/packages/Npgsql.3.2.3/lib/net451/Npgsql.xml
+/packages/Npgsql.3.2.3/lib/net45/Npgsql.xml
```

Comment:
Instead of ignoring individual files in the packages directory, you can just ignore the entire directory.